functional-tests/testdata/templating_helpers.go

```diff
@@ -0,0 +1,190 @@
+package testdata
+
+var TemplatingHelpers = `{
```

Comment:
I don't think the test data is being used.

Reply:
https://github.com/SpectoLabs/hoverfly/blob/templating-request-body/functional-tests/core/ft_templated_response_test.go#L120